CHANGELOG.md

```diff
@@ -90,6 +90,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (app) [\#617](https://github.com/cosmos/ethermint/issues/617) Fix genesis export functionality.
 * (rpc) [\#574](https://github.com/cosmos/ethermint/issues/574) Fix outdated version from `eth_protocolVersion`.
 
+## [v0.3.1] - 2021-02-25
```

Comment:
why are you releasing v0.3.2 instead of v0.4.1? It doesn't make sense to have release with a single fix and not include the fixes from above and the non-breaking improvements

Reply:
thanks for the question. this is just a quick fix that we needed to implement for a testnet issue we are encountering for the current version of the sdk (0.39.2). like you say, we are looking to release a v0.4.1 with the stargate fixes but this wont be done for some time. this is a patch to get the testnet back up in the mean time

Reply:
>  This is just a quick fix that we needed to implement for a testnet issue we are encountering for the current version of the sdk (0.39.2)

You should cherry-pick all the fixes and non-breaking changes to the latest Ethermint version instead. I also don't understand why are you running a testnet with an old version instead of the latest one.

>  like you say, we are looking to release a v0.4.1 with the stargate fixes but this wont be done for some time. this is a patch to get the testnet back up in the mean time

Stargate release is breaking so it should be v0.5.0